uflash.py

```diff
@@ -215,15 +269,23 @@ def main(argv=None):
         if arg_len == 0:
             flash()
         elif arg_len >= 1:
-            if argv[0] == 'help':
```

Comment:
I'm not particularly keen on this construction, but kept it to reduce the interface changes this test needed.
It's also slightly inconsistent at the moment in that `uflash help` will only print _HELP_TEXT, while `uflash -h` or --help will print the proper generated argparse help with usage etc. (that could probably be fixed quite easily).

I'd prefer to drop the `uflash help` construct unless you're particularly attached to it @ntoll